nixos/tests/rabbitmq.nix

```diff
@@ -19,5 +22,6 @@ import ./make-test-python.nix ({ pkgs, ... }: {
     machine.wait_until_succeeds(
         'su -s ${pkgs.runtimeShell} rabbitmq -c "rabbitmqctl status"'
     )
+    machine.wait_for_open_port("15672")
```

Comment:
Does this part of the test use the management plugin?

Reply:
the management plugin is just a UI that runs on that port. I thought that this would be a good way to test that at least it's running.